eos-tech-support/eos-stage-flatpak

```diff
@@ -69,6 +69,23 @@ for remote in `flatpak remote-list --system | awk {'print $1'}` ; do
         echo "Setting flatpak $remote URL to $new_url"
         flatpak remote-modify "$remote" --url="$new_url"
 
+        # Update the collection ID if it's set
+        collection_id=$(ostree --repo="$flatpak_repo" config get 'remote "'"$remote"'".collection-id' 2>/dev/null || echo "")
+        if [ -n "$collection_id" ] ; then
+            if [ "$STAGE" == "prod" ] ; then
+                new_collection_id=`echo "$collection_id" | sed -e "s/Dev.//" | sed -e "s/Demo.//"`
+            elif [ "$STAGE" == "dev" ] ; then
+                new_collection_id=`echo "$collection_id" | sed -e "s/endlessm./endlessm.Dev./"`
```

Comment:
This does not quite work for the swtich from demo to dev and vice-versa.  E.g.:
```
$ echo com.endlessm.Demo.flatpak | sed -e "s/endlessm./endlessm.Dev./"
com.endlessm.Dev.Demo.flatpak
```

Reply:
Oops, good catch